.gitlab/ci/global.yml

```diff
@@ -96,16 +97,24 @@
     - rm -rf $ARTIFACTS_FOLDER/logs
     - mkdir -p $ARTIFACTS_FOLDER/logs
     - section_end "Creating new clean logs folder"
+    - source $NVM_DIR/nvm.sh
+    - nvm use default
+    - npm ci --cache .npm --prefer-offline
+    - npm install jsdoc-to-markdown@5.0.3 -g  # disable-secrets-detection
```

Comment:
Can we make this its own section?

Reply:
Also, it may be helpful/relevant to additionally log the output to a logs file. I also think it may be useful to add a condition to test and see if `nvm` is installed (something like `if test $(command -v nvm); then ...`) and if not, add informative messages etc. etc. if you know what I mean - do you agree?

Reply:
I have a few questions about this section,
1. can't we have the image so that nvm is installed and sourced upon new shell instantiation (so we don't need the line `source $NVM_DIR/nvm.sh`)?
2. do you have it in the docker image so that `nvm` is configured so that `.nvrmc` files are respected as documented [here](https://github.com/nvm-sh/nvm#bash)? I ask cause it would make its usage more flexible for different repos.
3. why do we need to install `jsdoc-to-markdown` explicitly here? If it's something needed globally, should we not install it in the docker image already?

Reply:
0. About your second comment: I think if we take into account a case where someone changes the image to an image that doesn't use nvm it will either fail (if we leave it like this) or output a message and continue (if we change it to what you suggested). Since the nvm is installed so other repos could use this image more easily, I would rather it fail and make the developer think about the changes. 

1. I actually think you are right and we don't need this line anymore. Will push without it.

2. If you're talking about the bash script being inside the .bashrc - then no. Since we only use one nvm version, I see no point in guessing what other repos will need when installing other versions, especially since nvm.sh provides various functions to deal with special needs like ```nvm_find_nvmrc()``` for example. 

3. We need jsdoc-to-markdown installed explicitly since we use the ```jsdoc2md``` CLI command. In order for a CLI command to be installed properly through a node module installation it needs the "-g" flag which means its installed globally regardless of the package.json file (which contains the packages that will be installed in the npm ci command). 
Since other repos using this image could use different versions of jsdoc-to-markdown and might have no need for the ```jsdoc2md``` CLI command, I think it's better if it is installed explicitly like this.

Reply:
> 2
I don't follow your response. Let's resolve over slack.
> 3
Good point. 👍🏼